recipes/coverage-fixpaths/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "coverage-fixpaths" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18880405439c1a6329e5448b104c2b03de248fa4f54a7ec2f123fd226a2a573b
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
please add the python entry point:

https://github.com/omegacen/coverage-fixpaths/blob/master/setup.py#L23
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
Done. Good catch btw!